### DISCUSSION TITLE
Can we have ObjectId in array of embeddables

### DISCUSSION THREAD

**Discussion by 47vigen** [mikro-orm/mikro-orm#2716] [Q&A] [answered]
I use mongo driver and mikro-orm v5
Can we have ObjectId in array of embeddables?

**Comment by B4nan** [comment, marked as answer]
You can have array property there, but not a relation. In other words you can have array of anything, but you won't be able to populate it as if it was an entity relation.

```ts
@Property()
arr?: ObjectId[];
```

Collections are not supported in embeddables.
